c++/.bazelrc

```diff
@@ -6,6 +6,12 @@ common --enable_platform_specific_config
 common --enable_workspace
 common --incompatible_autoload_externally=
 
+# Debug build configuration with maximum runtime checks
+build:dbg -c dbg --//src/kj:debug_memory=True
```

Comment:
since this is a library these matter only for us.

Reply:
It's actually pretty reasonable to assume that bazel users compile their whole project with consistent flags so this actually seems fine for everyone. It's more the autotool users who will probably build opt builds and then link against them from debug builds.